### PR TITLE
Getters for struct members

### PR DESCRIPTION
**Pull request by marxriedl**

### CODE REVIEW

File: cmd/go-clang-generate/main.go [hunk 1]
```diff
@@ -201,7 +201,7 @@ func main() {
 	clangIndexHeaderFilepath := "./clang-c/Index.h"
 	tu := idx.Parse(clangIndexHeaderFilepath, []string{
 		"-I", ".", // Include current folder
-		"-I", "/usr/local/lib/clang/3.4.2/include/", // Include clang headers TODO make this generic
+		"-I", "/usr/include/clang/3.6.2/include/", // Include clang headers TODO make this generic
```
Comment (zimmski) [inline cmd/go-clang-generate/main.go]: Please add your include folder additionally to the existing include folder. I will figure out later how to resolve this, but removing mine breaks the generation on my side.

Comment (marxriedl) [inline cmd/go-clang-generate/main.go]: sorry, i actually wanted to revert this change

File: cmd/go-clang-generate/main.go [hunk 2]
```diff
@@ -427,6 +427,26 @@ func main() {
 			}
 		}
 
+		for _, s := range structs {
+			fmt.Println(s.Name + "blub")
```
Comment (zimmski) [inline cmd/go-clang-generate/main.go]: Remove debugging output

File: cmd/go-clang-generate/function.go [hunk 3]
```diff
@@ -500,27 +501,61 @@ func generateFunctionStructMemberGetter(f *Function) string {
...
+		sc = append(sc, {{if eq $.ArrayDimensions 2}}&{{end}}{{$.ElementType}}{{if $.IsPrimitive}}({{if eq $.ArrayDimensions 2}}*{{end}}goslice[is]){{else}}{{"{"}}{{if eq $.ArrayDimensions 2}}*{{end}}goslice[is]{{"}"}}{{end}})
 	}
 
 	return sc
 }
 `))
 
+/*
+
+	p := {{$.Receiver}}.c.{{$.Member}}
```
Comment (zimmski) [inline cmd/go-clang-generate/function.go]: Remove the comment for the commit.

File: cmd/go-clang-generate/main.go [hunk 4]
```diff
@@ -427,6 +427,26 @@ func main() {
...
+			fmt.Println(f.Receiver.Type.Name)
+			if s.Name == f.Receiver.Type.Name {
+				idx := -1
+				for i, mem := range s.Methods {
+					fmt.Println(mem)
```
Comment (zimmski) [inline cmd/go-clang-generate/main.go]: Remove debugging output

File: comment_gen.go [hunk 5]
```diff
@@ -23,46 +27,6 @@ func (c Comment) Child(ChildIdx uint16) Comment {
 	return Comment{C.clang_Comment_getChild(c.c, C.uint(ChildIdx))}
 }
 
-// A \c CXComment_Paragraph node is considered whitespace if it contains only \c CXComment_Text nodes that are empty or whitespace. Other AST nodes (except \c CXComment_Paragraph and \c CXComment_Text) are never considered whitespace. \returns non-zero if \c Comment is whitespace.
-func (c Comment) IsWhitespace() bool {
```
Comment (zimmski) [inline comment_gen.go]: There is something wrong. There should never be a removal of existing generations, unless we generated them wrong in the first place.
